src/page-multi-facility/HistoricalCasesChart/index.tsx

```diff
@@ -23,14 +26,20 @@ function generateBarChartData(
     });
 
     if (existingVersion) {
+      const cases = totalConfirmedCases(existingVersion);
+      // This is necessary since semiotic doesn't have an option to display a stacked bar chart overlay.
+      // So we display population (non-cases) = total population - cases
```

Comment:
I think this behavior is causing strange behavior. One case I have found: if population is not filled in for a given day, the bar will go below the Y-axis. To test, you can add a historical case on a date that looks something like this:

<img width="1792" alt="Screen Shot 2020-05-25 at 23 27 49" src="https://user-images.githubusercontent.com/431895/82863423-5fc71680-9edf-11ea-8665-ea051e6740df.png">

You'll see the bar look like this, stretching into negative:

<img width="576" alt="Screen Shot 2020-05-25 at 23 26 50" src="https://user-images.githubusercontent.com/431895/82863438-6bb2d880-9edf-11ea-9881-6f629f4ceea5.png">

Reply:
to be fair, this is really the fault of our failing to validate the form inputs correctly; having a case count larger than a population count (in this case, zero) is nonsense data. It probably ought to be handled before it ever gets to the chart. The chart's behavior is correct when the input data doesn't violate the assumptions of what it's supposed to represent

Reply:
Yeah, I figured we should never have a scenario where the total population is less than the case count.
Since we don't have input validation as of now, I'll handle this in code.